pkg/components/imguploader/imguploader.go

```diff
@@ -76,6 +76,20 @@ func NewImageUploader() (ImageUploader, error) {
 		path := gcssec.Key("path").MustString("")
 
 		return NewGCSUploader(keyFile, bucketName, path), nil
+	case "local":
```

Comment:
Im not sure how this will work. Whats the purpose of the public path? How Should the user access the images over http? 

By default grafana saves rendered images to `data/png/` cant we just reuse that path?

Reply:
The idea was to make it configurable for arbitrary use-cases and just make it work for now.

Some reasoning:
- I didn't know how exactly the `data` directories are supposed to be used and I didn't want to actually expose them to the public frontend user as it seems logical that these are used by the backend.
- I was not sure if implementing the logic using `http_server` static routes or similar is the way to go.
- As `imageOnDiskPath` is passed, I can't assume `data/png` as the image path. Managing routes per passed images is also not an option.

So the simple solution was to just copy them to a custom folder that is accessible by the frontend.

My example configuration is this:
```
[external_image_storage.local]
path = ../data/storage
public_path = public/img/attachments
```

Do you have a suggestion how to prevent the file duplication here or if adding a static route to `data/png` is the way to go? I'd assume the optimal implementation would just expose the path with the generated images, not copy anything and not need any configuration.

Reply:
Adding a route to the data/png should be enough. Using local image store should be a configurationless feature.